ios/browser/api/certificate/utils/brave_certificate_utils.mm

```diff
@@ -0,0 +1,74 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/certificate/utils/brave_certificate_utils.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+namespace brave {
+NSData* string_to_data(const std::string& str) {
+  return str.empty()
+             ? [[NSData alloc] init]
+             : [NSData dataWithBytes:reinterpret_cast<const std::uint8_t*>(
+                                         str.c_str())
+                              length:str.size()];
+}
+
+NSDate* date_to_ns(double time_interval) {
+  return [NSDate dateWithTimeIntervalSince1970:time_interval];
+}
+
+NSDate* date_to_ns(const base::Time& date) {
+  return [NSDate dateWithTimeIntervalSince1970:date.ToDoubleT()];
+}
```

Comment:
Can be removed, use [`ToNSDate()`](https://source.chromium.org/chromium/chromium/src/+/main:base/time/time.h;l=693?q=time.h) on `base::Time`